scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -57,13 +57,16 @@ const volatile bool kthreads_local = true;
 const volatile bool max_dsq_pick2 = false;
 const volatile bool select_idle_in_enqueue = true;
 
+const volatile u64 lb_slack_factor = 5;
```

Comment:
Is it worth making this a 128th instead of a percentage so we divide by a power of 2? Now the load balancing path is in dispatch it's not that hot and it likely doesn't make a big difference, but this is a pattern we've used elsewhere.